rootfs/api/models.py

```diff
@@ -326,7 +326,10 @@ def scale(self, user, structure):  # noqa
     def _scale_containers(self, scale_types, to_remove):
         release = self.release_set.latest()
         for scale_type in scale_types:
-            image = release.image
+            if release.build.dockerfile:
+                image = release.image
+            else:
+                image = release.image.split(":")[0]+":git-"+release.build.sha
```

Comment:
this is the kind of information that builder should push up to the controller on a push hook (I believe that's how it's done today IIRC). Otherwise this will break `deis pull`.